### DISCUSSION TITLE
Config of HRM System default time for US Central Standard Time (CST)

### DISCUSSION THREAD

**Discussion by umairkhanrind** [horilla-opensource/horilla#268] [General]
I've recently started using your hrm, and it appears to be much better than other HRM software I've tried. However, I have a question. My team is based in Pakistan, but they are working for a US-based company. My CEO wants all employee attendance to be recorded in US Central Standard Time (CST) instead of Pakistan Standard Time (PST). Is it possible to configure the HRM system to reflect or configured to US time zone, so that when employees mark their attendance, it shows the time in CST instead of PST?

**Comment by horilla-opensource** [comment]
> I've recently started using your hrm, and it appears to be much better than other HRM software I've tried. However, I have a question. My team is based in Pakistan, but they are working for a US-based company. My CEO wants all employee attendance to be recorded in US Central Standard Time (CST) instead of Pakistan Standard Time (PST). Is it possible to configure the HRM system to reflect or configured to US time zone, so that when employees mark their attendance, it shows the time in CST instead of PST?

You can change the time settings from the settings.py file (`horilla/settings.py`) to your preferred time zone and Horilla will make use of it for all its computations.

With Regards,
**Team Horilla**
